blue-common/src/main/resources/application.conf

```diff
@@ -27,7 +27,7 @@ blue {
   #    then receives an information email with summary of registration
   #   `no-email` the user chooses his password upon registration and receives no
   #   email
-  registration-confirmation = "email-confirmation"
+  registration-confirmation = "no-email"
```

Comment:
Yes, I know that's wrong, it's due to the old organization...

The real distribution uses the configuration from the blue-dist project (https://github.com/gnieh/bluelatex/blob/master/blue-dist/src/configuration/org.gnieh.blue.common/application.conf).

The file in `src/main/configuration` is only used for test instances.